src/docdb/docDBConnectionStrings.ts

```diff
@@ -10,7 +10,7 @@ export function parseDocDBConnectionString(connectionString: string): ParsedDocD
     const endpoint = getPropertyFromConnectionString(connectionString, 'AccountEndpoint');
     const masterKey = getPropertyFromConnectionString(connectionString, 'AccountKey');
     const databaseName = getPropertyFromConnectionString(connectionString, 'Database');
-    if (!endpoint || !masterKey) {
+    if (!endpoint || !masterKey || connectionString.match(" ")) {
```

Comment:
Can you point to documentation that indicates spaces inside a connection string are not allowed?  For instance, what if a password has spaces?